### PR TITLE
Fix std json static assert

### PR DESCRIPTION
**Pull request by atilaneves**

### CODE REVIEW

File: std/math/operations.d [hunk 1]
```diff
@@ -1717,7 +1717,7 @@ if (isFloatingPoint!T)
     bool negative;
 }
 
-FloatingPointBitpattern!T extractBitpattern(T)(T val)
+FloatingPointBitpattern!T extractBitpattern(T)(T val) @trusted
```
Comment (thewilsonator) [inline std/math/operations.d:1720]: I still think that this should use `@trused` lambdas but as this is a commit that is from stable, I'll let this go.

**Review by thewilsonator — APPROVED** [review]

**Comment by RazvanN7** [comment]
You might want to rebase this as it contains some previously merged commits.

**Comment by thewilsonator** [comment]
IIRC those were to stable

**Comment by RazvanN7** [comment]
My bad. So do we want them in master also?

**Comment by thewilsonator** [comment]
Well we should probably just merge stable instead.

**Comment by ibuclaw** [comment]
2.097 is out, hasn't the merge happened yet?

**Comment by atilaneves** [comment]
The failures seem completed unrelated.

**Comment by thewilsonator** [comment]
I had those errors on a recent dmd PR, try rebasing.

**Comment by atilaneves** [comment]
Errors again. I rebased on upstream/master since I don't really want to deal with this.

**Pull request merged**